### PR TITLE
Kerpecheva/remove token command

### PR DESCRIPTION
**Pull request by kerpecheva**

### CODE REVIEW

File: controls/combobox/commands.md [hunk 1]
```diff
@@ -8,11 +8,15 @@ slug: combobox-commands
...
 
-The [.NET MAUI ComboBox]({%slug combobox-overview%}) provides the following commands:
+The [.NET MAUI ComboBox]({%slug combobox-overview%}) provides a few commands that will help you manipulate its selection.
+
+## SelectAll Command
 
 - `SelectAllCommand` (`ICommand`)&mdash;Selects all items from the source.
 
-> The `SelectAll` command can be used only when `SelectionMode` is `Multiple`. An exception will be thrown if the command is invoked in `Single` `SelectionMode`.
+> The `SelectAll` command can be used only when `SelectionMode` is `Multiple`. In case the command is invoked in `Single` `SelectionMode`, the ComboBox throws an exception.
```
Comment (yordan-mitev) [inline controls/combobox/commands.md:17]: - Avoid the passive voice.
- Avoid the visual clutter caused by the close placement of different styles (headings, inline code, a note)&mdash;as the section contains only a single line of text (except the content of the note), you don't need to use the note style to grab the attention of the reader. 
```suggestion
To use the `SelectAll` command, set the selection mode to `Multiple`. If you invoke the command in `Single` selection mode, the ComboBox throws an exception.
```

File: controls/combobox/commands.md [hunk 2]
```diff
@@ -8,11 +8,15 @@ slug: combobox-commands
...
+
+## Clear Selection Command
 
 - `ClearSelectionCommand` (`ICommand`)&mdash;Sets the selection to null. If Multiple `SelectionMode` is used, this command will clear all selected items. YOu can override the default behavior and create custom command.
```
Comment (yordan-mitev) [inline controls/combobox/commands.md:21]:
```suggestion
`ClearSelectionCommand` (`ICommand`)&mdash;Sets the selection to null. If you use the `Multiple` selection mode, this command will clear all selected items. You can override the default behavior and create a custom command.
```

File: controls/combobox/commands.md [hunk 3]
```diff
@@ -24,6 +28,32 @@ The example below shows both cases, the default `ClearSelectionCommand` executio
 
 ![ComboBox ClearSelectionCommand](images/combobox-commands.gif)
 
+## Remove Token Command
```
Comment (yordan-mitev) [inline controls/combobox/commands.md:31]: The suggestions from the `controls/autocomplete/commands.md` article apply here too.

File: controls/autocomplete/commands.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+description: Learn more about the commands that the Telerik UI for .NET MAUI AutoComplete control exposes.
+position: 14
+slug: autocomplete-commands
+---
+
+# .NET MAUI AutoComplete Commands
+
+The .NET MAUI AutoComplete provides a `RemoveTokenCommand` that will help you manipulate its tokens selection.
+
+## Remove Token Command
```
[the remark this answers is not in the document]
Comment (kerpecheva) [reply controls/autocomplete/commands.md:13]: we'll add more commands for the release, that's why I added this as a section

**Pull request merged**